tests/Integration/Behaviour/Features/Context/Domain/Product/UpdatePackFeatureContext.php

```diff
@@ -94,12 +95,17 @@ public function assertPackEmpty(string $packReference): void
     {
         $packId = $this->getSharedStorage()->get($packReference);
 
-        $packedProducts = $this->getQueryBus()->handle(new GetPackedProducts($packId));
+        $packedProducts = $this->getQueryBus()->handle(
+            new GetPackedProductsDetails(
+                new ProductId($packId),
```

Comment:
We use primitive types for our constructor Although we did discuss this with @saulaski wondering if we should rely on VO instead But until an architectural decision is made to do it differently let's follow the current convention and use primitive types, so integers here

Reply:
yes, It's ok with the previous message about that